src/nvim/tui/tui.c

```diff
@@ -1752,6 +1757,44 @@ static int unibi_find_ext_bool(unibi_term *ut, const char *name)
   return -1;
 }
 
+/// Determine if the terminal supports truecolor or not:
+///
+/// 1. If $COLORTERM is "24bit" or "truecolor", return true
+/// 3. Else, check terminfo for Tc, RGB, setrgbf, or setrgbb capabilities. If
+///    found, return true
+/// 4. Else, return false
```

Comment:
Looks like "2." is missing :) Should we mention `_tui.lua` / `_tty.lua` here?

Reply:
Fixed, thanks